website/pages/en/network/curating.mdx

```diff
@@ -31,6 +31,8 @@ In general, a bonding curve is a mathematical curve that defines the relationshi
 
 In the case of The Graph, [Bancor’s implementation of a bonding curve formula](https://drive.google.com/file/d/0B3HPNP-GDn7aRkVaV3dkVl9NS2M/view?resourcekey=0-mbIgrdd0B9H8dPNRaeB_TA) is leveraged.
 
+> **Note**: On Arbitrum, the bonding curve dynamics is a little different than on Ethereum mainnet. On L2 the price per share signal does not change over time.
```

Comment:
I think "over time" is a bit confusing, maybe just "On L2 the price per share signal does not change (i.e. the bonding curve is flat)."

Reply:
Okay, done.